neuvector-agent.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: neuvector-agent
+  version: 5.3.2
+  epoch: 0
+  description: "NeuVector Agent"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: edbdcba632835d56dcc92ba86323c8a196471289
+      repository: https://github.com/neuvector/neuvector.git
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: github.com/docker/docker@v26.1.3+incompatible github.com/containerd/containerd@v1.5.18 github.com/opencontainers/runc@v1.1.12 github.com/opencontainers/image-spec@v1.1.0 github.com/docker/distribution@v2.8.2-beta.1+incompatible golang.org/x/sys@v0.19.0 golang.org/x/net@v0.23.0 golang.org/x/text@v0.14.0 gopkg.in/yaml.v3@v3.0.1
+      replaces: github.com/samalba/dockerclient=github.com/Dentrax/dockerclient@v0.1.0
```

Comment:
is this a right module that you want to replace with? @Dentrax you should take a look at here 

github.com/Dentrax/dockerclient@v0.1.0

Reply:
Yup. Dockerclient is no longer maintained. @Dentrax resolved a CVE by merging a change that bumped jsonlog: https://github.com/samalba/dockerclient/tree/a30362618471

The Docker management engine API is meant to replace this but would require extensive changes